test/verify/check-shell-multi-machine

```diff
@@ -818,6 +818,7 @@ class TestMultiMachine(MachineCase):
 
         # Relogin.  This should now work seamlessly.
         b.relogin(None, wait_remote_session_machine=m1)
+        self.allow_restart_journal_messages()
```

Comment:
Just an idea - why don't we add this into definition of relogin/logout/reload...?

Reply:
Since c3309b7a56 `Browser` has reference to `machine`. I might take a look at this.

Reply:
I stand corrected -- last time I checked this wasn't the case, and it felt a little dirty (layering violation). But so much the better, then we can fix this wholesale :partying_face: thanks!